### ISSUE TITLE
Feature: Basic insight

### ISSUE THREAD

**Issue by oseasmoran73** [januschung/job-winner-ui#18]
## Description

It would be nice to have some basic analytics on the amount of applications applied, interviewed, and rejected.

**Comment by januschung** [comment]
I agree. My plan was to use those dummy icons on the top right corner to show the count for `open` application and `open` interview and `open` offer.

**Comment by januschung** [comment]
> I agree. My plan was to use those dummy icons on the top right corner to show the count for `open` application and `open` interview and `open` offer.

This part is done

**Comment by januschung** [comment]
introduced a status bar to show count of job appilcation applied and different status
